apps/mark-scan/frontend/src/pages/poll_worker_screen.tsx

```diff
@@ -260,40 +260,44 @@ export function PollWorkerScreen({
 
     if (stateMachineState === 'waiting_for_ballot_data') {
       return (
-        <Screen>
-          <Main centerChild padded>
-            <Prose>
-              <FullScreenIconWrapper align="center">
-                <Icons.Done color="success" />
-              </FullScreenIconWrapper>
-              <H2 as="h1" align="center">
-                {`Voting Session Active: ${ballotStyleId} at ${precinct.name}`}
-              </H2>
-              <p>Paper has been loaded.</p>
-              <ol style={{ marginBottom: '0' }}>
-                <li>
-                  <P>
-                    Instruct the voter to press the{' '}
-                    <Font weight="bold" noWrap>
-                      Start Voting
-                    </Font>{' '}
-                    button on the next screen.
-                  </P>
-                </li>
-                <li>
-                  <P>Remove the poll worker card to continue.</P>
-                </li>
-              </ol>
-              <HorizontalRule>or</HorizontalRule>
-              <P align="center">Deactivate this voter session to start over.</P>
-              <P align="center">
-                <Button onPress={resetCardlessVoterSession}>
-                  Deactivate Voting Session
-                </Button>
+        <CenteredCardPageLayout
+          buttons={
+            <Button onPress={resetCardlessVoterSession}>
+              Deactivate Voting Session
+            </Button>
+          }
+          icon={<Icons.Done color="success" />}
+          title="Voting Session Active:"
+          voterFacing={false}
+        >
+          <H3 as="h2">
+            <Font weight="regular">
+              Ballot Style {extractBallotStyleGroupId(ballotStyleId)} at{' '}
+              {precinct.name}
+            </Font>
+          </H3>
+          <p>Paper has been loaded.</p>
```

Comment:
`<p`> is my old code, but should this be `<P>`?

Reply:
+1 - thanks for catching!